docs/guides/go-built-in.md

```diff
@@ -311,112 +310,95 @@ information on why the transaction was rejected.
 
 Note that `CheckTx` does not execute the transaction, it only verifies that the transaction could be executed. We do not know yet if the rest of the network has agreed to accept this transaction into a block.
 
-
-Finally, make sure to add the bytes package to the `import` stanza at the top of `app.go`:
+Finally, make sure to add the `bytes` package to the `import` stanza at the top of `app.go`:
 
 ```go
 import(
-	"bytes"
+    "bytes"
 
-	"github.com/dgraph-io/badger/v3"
-	abcitypes "github.com/cometbft/cometbft/abci/types"
+    "github.com/dgraph-io/badger/v3"
+    abcitypes "github.com/cometbft/cometbft/abci/types"
 )
 ```
 
+### 1.3.3 FinalizeBlock
 
-### 1.3.3 BeginBlock -> DeliverTx -> EndBlock -> Commit
+`FinalizeBlock` is an ABCI method introduced in CometBFT `v0.38.0`. This replaces the functionality provided previously (pre-`v0.38.0`) by the combination of ABCI methods `BeginBlock`, `DeliverTx`, and `EndBlock`.
```

Comment:
```suggestion
When the CometBFT consensus engine has decided on the block, the block is transferred to the
application via `FinalizeBlock`.
`FinalizeBlock` is an ABCI method introduced in CometBFT `v0.38.0`. This replaces the functionality provided previously (pre-`v0.38.0`) by the combination of ABCI methods `BeginBlock`, `DeliverTx`, and `EndBlock`.
```

Reply:
I must insist that I don't see the point of mentioning the old methods here. This is an introductory guide, for people that don't know the older versions.
Upgrading from older versions should be a tutorial in itself.

Reply:
I see @lasarojc. But I don't really see any harm in mentioning previous methods. If a legacy user from 2019 or 2020 comes in he will get some context on what happened to old methods. New users, they might just not care and move ahead.